app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2445,7 +2547,7 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     }
 
 
-    // === Add Node ===
+    // === Nodes Utilities ===
```

Comment:
"Utilities" is the worst word IMO, as everything can be an "utility". This section will probably contain everything in the future. Do you see any better name maybe?

Reply:
Ok, I read it more carefully, and it looks like things related to mouse cursor and clicks. So I rename it to "Mouse Interactions"